src/signal_manager.sh

```diff
@@ -0,0 +1,30 @@
+. "$KW_LIB_DIR/kwio.sh" --source-only
```

Comment:
@rodrigosiqueira yes, I can confirm this will be fixed by #240. Actually this error is what originally motivated me to implement the include guards haha

Reply:
@joaoseckler, I finally applied the include feature. Could you update your PR based on that? Also, could you check if this error gets fixed?